src/pages/test-report/test-report.ts

```diff
@@ -84,14 +84,22 @@ export class TestReportPage extends BasePageComponent {
     };
 
     const { candidateUntitledName$, isSeriousMode$, isDangerousMode$ } = this.pageState;
-
     const merged$ = merge(
       candidateUntitledName$,
       isSeriousMode$.pipe(map(result => this.isSeriousMode = result)),
       isDangerousMode$.pipe(map(result => this.isDangerousMode = result)),
     );
-
     this.subscription = merged$.subscribe();
+
+    this.manoeuvres$ = this.store$.pipe(
+      select(getTests),
+      select(getCurrentTest),
+      select(getTestData),
+      select('manoeuvres'),
```

Comment:
I don't think we use the string literal pattern elsewhere (I didn't even realise that it was possible). Probably best to convert into a selector to match the rest of the codebase.

Reply:
yeah I did wonder if I should do that. you can do select('manoeuvres','selectedReveseParking'] etc